src/metabase/util/memoize.cljs

```diff
@@ -0,0 +1,130 @@
+(ns metabase.util.memoize
```

Comment:
As noted above, this should have a companion `.clj` file that re-exports a few functions from `clojure.core.memoize`, so the two files have the same API. Then this namespace can be `:require`d naively and will just work on both platforms.

Reply:
done